epaint/src/tessellator.rs

```diff
@@ -192,6 +192,117 @@ pub mod path {
 
     use super::*;
 
+    mod precomputed_quadrants {
+        use emath::{vec2, Vec2};
+
+        pub const QUADRANTS_0_5: [[Vec2; 3]; 4] = [
```

Comment:
to silence the clippy warnings:

```suggestion
        #[allow(clippy::approx_constant)]
        pub const QUADRANTS_0_5: [[Vec2; 3]; 4] = [
```